composer.json

```diff
@@ -24,11 +24,12 @@
 		"php": "^5.6 || ^7.0 || ~8.0.0 || ~8.1.0 || ~8.2.0",
 		"ext-gd": "*",
 		"ext-mbstring": "*",
+		"chromatic/psr-log-aware-trait": "^2.0 || ^3.0",
```

Comment:
Would you mind the package be created under mpdf organization and namespace?

Reply:
That is fine with us. Would you like us to transfer the repository?

Reply:
If you can, that'd be great.

Reply:
@finwe I tried to initiate the transfer but received the following error:

> You don’t have the permission to create public repositories on mpdf

Do you want to temporarily give me access to the org or would you rather just clone the repo and push it up to your org?